### ISSUE TITLE
support mixing top-level fields from two different schemas

### ISSUE THREAD

**Issue by ecthiender** [hasura/graphql-engine#1371]
After a remote schema is added, currently Hasura does not allow mixing root fields from different schemas. Add support for it.

E.g:

```graphql
query {
  profile {
    id
    name
  }
  weather(city: "BLR") {
    temperature
  }
}
```

Here `profile` is from Hasura schema (a table). And `weather` is from a remote schema.

**Comment by bkstorm** [comment]
Yeah, just encounter this issue, hope hasura will support it.

**Comment by karibertils** [comment]
encountered this as well, hope it gets supported soon !

**Comment by 0x777** [comment]
Depends on #2801

**Comment by havsar** [comment]
is there an ETA for this?

**Comment by abooij** [comment]
Closed by #5869.
